usl_pipeline/usl_pipeline/formats/geotiff_loader.py

```diff
@@ -0,0 +1,39 @@
+import rasterio.features
+
+
+def load_elevation_from_geotiff(
+    file_path: str, load_data: bool = False, result_nodata: float = None
+):
+    """
+    Loading elevation raster data from GeoTIFF file.
+    :param file_path: path to a file to load from
+    :param load_data: indicates that in addition to header, data should also be
```

Comment:
I'd expect a function named 'load_elevation_from_geotiff' to load the elevation data by default. I'd either
- have a separate 'read_tiff_header' function for callers to use if they only want the header
- name the parameter 'header_only' and have the default be false

Reply:
I went with 'header_only=false' option.